### DISCUSSION TITLE
Looking for group of begginig js students

### DISCUSSION THREAD

**Discussion by Bartroz** [codewars/codewars.com#2901] [General]
Hello
I've started learn js and I'm looking for people who'd like to create group with me and discuss about problems during completing kata :)

**Comment by wrp40** [comment]
same

**Reply by Bartroz** [reply]
Hello, still interested? Maybe at first we can try if this make sense

**Reply by Raygbrains22** [reply]
Interested

**Comment by Rickyngechu** [comment]
Hello ,I'm willing to join you guys in this js journey.

**Comment by Bartroz** [comment]
Okay, so maybay we should set up a meeting on ds? I can start a channel and add you

**Reply by Rickyngechu** [reply]
Sure ,create one send an invite link to me .Or let me know if I'm to share my username to you.

**Comment by darihoney** [comment]
Hi guys! Can I join to you?

**Reply by Rickyngechu** [reply]
Sure, lets wait for the discord group to be created.

**Comment by Bartroz** [comment]
Hello everyone!
Sorry for my absence but I had personal issues.  Right now we can start a coorperation. Give me your ds names co I can invite you

**Reply by Rickyngechu** [reply]
Ricky.exe#4697

**Reply by darihoney** [reply]
darihoney#3262

**Reply by WokeCat** [reply]
Hi, I'm interested in joining too if you're still open to adding more people to the group!
SleepyCat#2547

**Comment by HasanAlilhadi** [comment]
Hey, can I join ?

**Comment by Bartroz** [comment]
Yeah, sure - everyone is welcomed
Below i insert the link to the server\
https://discord.gg/vmP9vvAC

**Comment by SkAliya** [comment]
> Hello I've started learn js and I'm looking for people who'd like to create group with me and discuss about problems during completing kata :)

hey hii ,im also

**Reply by PoohJr** [reply]
Same here, swing that invite tryna learn some more too.

**Comment by zoeparker123** [comment]
For a beginner JavaScript group, a structured weekly plan could work well, choose one theme, solve a few Codewars challenges related to it, and then build a small project applying those concepts. For example, you could create a simple tool or [calculator](https://cgpa2percentageonline.com/) that fetches JSON data, displays categories, includes search functionality, and performs basic calculations or totals.

**Comment by hobovsky** [comment]
This type of discussion is off-topic for GitHub. We handle social and learning matters on our [Discord server](https://discord.gg/mSwJWRvkHA) and its `#ask-help` and `#beginner-intros` channels, where it’s easier, and safer for everyone to join public channels and learn together.

It’s much more effective (and fun!) to share progress and ask questions in open channels with man users rather than private groups. Please continue this conversation on Discord instead.

**Comment by jennaclarke875** [comment]
Collaborative learning can really help beginners stay consistent. I’ve been working on small JavaScript projects myself, even things like simple restaurant [menu pages](https://pappadeauxmenu.com/) or fitness tracking dashboards, and it’s amazing how much real-world examples help in understanding core concepts. 

Would love to connect and share progress with other learners here.
